src/components/Social/Facebook/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * react libraries
+*/
```

Comment:
For imports, you don't need to use block comments, use inline comments instead
```suggestion
// react libraries
```